kuyruk/task.py

```diff
@@ -194,6 +198,9 @@ def _module_name(self):
         return name
 
 
+SubTask = namedtuple("SubTask", "task, args, kwargs, host")
```

Comment:
No, they are the same. Just a different notation. Tuple looks better, I will update it.